src/routes/User/components/EditUser/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { editUser } from 'reducers/user';
```

Comment:
I feel like this is gonna get clunky very fast with new features being added. Maybe * as userActions ?